vignettes/fd_aboveground_vignette.Rmd

```diff
@@ -62,6 +62,8 @@ The `fd_inventory()` script within `fortedata` currently includes the following
 
 * `fd_subcanopy_density()` returns a single dataset of subcanopy (1-8cm DBH) stem density within one quarter (0.025 ha) of all 32 subplots in the FoRTE experiment. Stem density survey were conducted during the 2019 field season.
 
+* `fd_double_dendro()` returns a single dataset of dendrometer band measurements (in) at ~1.3m stem height and ~1.8m stem height on a subsample of trees >= 8cm DBH.

```

Comment:
Do we want to perform a conversion to SI units before returning to the user?

Reply:
@bpbond thanks! Yeah, glad it was a lighter load for you this time. Hmmm yeah it's something I have considered and it makes sense to me. The bands are built with inch ruler stickers so that's why they are in inches, but any and all analysis that I've done has always been in cm. Also, the subcanopy is reported in mm which could be quickly switched to cm if we are shooting for consistency.

Reply:
Great. I think the non-SI unit issue is the big one to address; consistent of mm versus cm up to you and not critical.